ddtrace/tracer/context_test.go

```diff
@@ -116,7 +116,7 @@ func Test128(t *testing.T) {
 	_, _, _, stop := startTestTracer(t)
 	defer stop()
 
-	os.Setenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", "false")
```

Comment:
Hmmm this doesn't seem like a full change? (I'm surprised this didn't break the test)

Reply:
Thanks for spotting that. I've split `Test128` into two subtests https://github.com/DataDog/dd-trace-go/pull/2437/commits/7dbf3362b856d34d8b409cd87e15bf797a1aca44.